docs/templates/README.md

```diff
@@ -0,0 +1,57 @@
+# WooCommerce Block Templates
+
+This page includes documentation related to WooCommerce Block Templates.
+
+## Table of Contents
+
+* [Overview](#overview)
+  * [Requirements](#requirements) 
+* [Technical Overview](#technical-overview)
+  * [The Problem](#the-problem)
+  * [The Solution](#the-solution)
+* [Some things to be aware of](#some-things-to-be-aware-of)
+* [Related Files](#related-files)
+
+## Overview
+
+At the highest level, WooCommerce Block Templates are a collection of WooCommere core templates that are available in the WordPress Full Site Editing feature introduced as part of WordPress 5.9. Enabling you to customize these templates with blocks via the Site Editor. 
+
+You can read more about the Full Site Editing (FSE) feature [here](https://developer.wordpress.org/block-editor/getting-started/full-site-editing/).
+
+### Requirements
+
+| Software        | Minimum Version  |
+|-----------------|------------------|
+| WordPress       | 5.9              |
+| WooCommerce     | 6.0              |
+| Block theme     | -                |
+
+## Technical Overview
+
+### The Problem
+
+The FSE feature does not accommodate for loading templates from plugins such as WooCommerce (or WooCommerce Blocks). Instead all template files loaded natively with this feature are to be present within the active themes files. This is a problem because we would like WooCommerce specific templates to be compatible with every theme, and to do this we have to provide themes that don't have WooCommerce specific templates with them from our plugin.
+
+### The Solution
+
+We created a custom solution which involves a handful of files (listed in the below table) that are responsible for both making the templates available as a block template, and rendering it on the front-end.
+
+The BlockTemplateController class is primarily responsible for hooking into both WordPress core, and WooCommerce core filters to load WooCommerce [templates](https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/trunk/templates/templates) in the Site Editor and on the front-end. It is in this class where the majority of the logic is handled.
+
+### Some things to be aware of
+
+* Individual templates are represented by a placeholder block within the Site Editor until we can 'block-ify' these. This is the long-term goal. Until then users will be able to customize templates by adding blocks above and below the placeholder.
```

Comment:
```suggestion
* Individual templates are represented in their entirety by a placeholder block within the Site Editor until we can cut them up into discrete blocks. This is our long-term goal. However, until then, users will be able to customize templates by adding blocks above, below and around the placeholder.
```

Reply:
Agree that 'block-ify' is unclear! That said, maybe `divide them up` instead of `cut them up` here? Might just be me but `cut them up` does feel like it could have a bit of a negative connotation. 😅

Also, I'd leave it at above and below, as around isn't really that much of an option in the current state.